.github/workflows/atlantis-image.yml

```diff
@@ -20,10 +23,25 @@ jobs:
         registry: ghcr.io
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
+
+    # Publish latest image to container registry
     - name: build atlantis:latest image
+      if: ${{ github.event_name == 'push'}}
       run: |
         make build-service
         docker build -t ghcr.io/runatlantis/atlantis:latest .
     - name: publish atlantis:latest image
+      if: ${{ github.event_name == 'push'}}
       run: |
         docker push ghcr.io/runatlantis/atlantis:latest
+
+    # Publish release to container registry
+    - name: build atlantis release image
```

Comment:
```suggestion
    - name: build atlantis:$GITHUB_REF release image
```

we can also do some fancy stuff in the step name

Reply:
Yeah i didn't know if this would work in the name itself.